### ISSUE TITLE
the accuracy is always 0

### ISSUE THREAD

**Issue by liwer0715** [hx173149/C3D-tensorflow#124]
i run without pre-trianing model ,the accuracy is 0 ,but using pre-training model,it is still 0

**Comment by MrLuer** [comment]
this project is rubbish, in chinese we call it 辣鸡

**Comment by MrLuer** [comment]
please see issue 25,[https://github.com/hx173149/C3D-tensorflow/issues/125](url)

**Comment by MrLuer** [comment]
please see #125
